### DISCUSSION TITLE
Can you paste a list of urls for VDH to bulk download?

### DISCUSSION THREAD

**Discussion by edwinicq** [aclap-dev/video-downloadhelper#883] [Help and Q&A]
### Your question

Can you paste a list of urls for VDH to bulk download (i.e., 1 video per url, and a list of 100 urls)?
And can VDH automatically choose the largest file size to download?

Thank you!

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "download_directory": "C:\\Users\\edwin\\Downloads",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 5,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I believe an idea like this has been floated here before.  Probably more than once.  It seems to me @paulrouget Paul (who is the developer) has discussed something like this & agreed it might be something he will consider doing at some future date.  But the product does not support this functionality at the moment.  I would probably use it.

**Reply by paulrouget** [reply]
Yes that's something we have in our todolist. No supported yet.
